carball/analysis/analysis_manager.py

```diff
@@ -177,20 +231,38 @@ def write_pandas_out_to_file(self, file):
     def get_protobuf_data(self) -> game_pb2.Game:
         """
         :return: The protobuf data created by the analysis
+
+        USAGE: A Protocol Buffer contains in-game metadata (e.g. events, stats)
+
+        INFO: The Protocol Buffer is a collection of data organized in a format similar to json. All relevant .proto
+        files found at https://github.com/SaltieRL/carball/tree/master/api.
```

Comment:
We should add that you can treat the protobuf object as a python object whose fields match the API